modules/filter_fastq.py

```diff
@@ -0,0 +1,71 @@
+from typing import Union, Tuple
+
+
+def filter_gc(seq: str, gc_bounds: Union[float, Tuple[float]]) -> bool:
+    '''
+    Checks if nucleic acid sequence fits the specified GC-content.
+
+    Arguments
+    ---------
+    seq (str): A nucleic acid sequence to be analyzed.
+
+    gc_bounds Union[float, Tuple[float]]: Bounds of accepted GC-content.
+    Can be given as single number (treated as upper limit) or as an
+    interval (tuple).
+
+    Returns
+    -------
+    bool: True if GC-content of sequence lies within the bounds.
+    False otherwise.
+    '''
+    if isinstance(gc_bounds, Union[float, int]):
```

Comment:
По идее isinstance напрямую поддерживает перебор возможных типов
```suggestion
    if isinstance(gc_bounds, (float, int)):
```

Reply:
Покрутил в Юпитере. И правда работает, спасибо!